proxy/http2/Http2Stream.h

```diff
@@ -334,9 +408,27 @@ Http2Stream::read_vio_writer() const
   return this->read_vio.get_writer();
 }
 
+inline bool
+Http2Stream::read_enabled() const
+{
+  return !this->read_vio.is_disabled();
+}
+
 inline void
 Http2Stream::_clear_timers()
 {
   _timeout.cancel_active_timeout();
   _timeout.cancel_inactive_timeout();
 }
+
+inline void
+Http2Stream::read_update(int count)
+{
+  read_vio.ndone += count;
+}
+
+inline void
+Http2Stream::read_done()
```

Comment:
It's unclear what will be done by this function. `read_enabled` returns whether it's enabled but `read_done` doesn't return whether it's done.

I'd name these functions `is_read_enabled` `update_read_length` and `set _read_done`

Reply:
Yes, this is good.